### ISSUE TITLE
Return string instead of md file

### ISSUE THREAD

**Issue by Sopralapanca** [ssine/pptx2md#71]
Is it possible to obtain a string in Markdown format instead of saving a .md file?

**Comment by adamcvolz** [comment]
Agreed, makes a lot of sense to get a text back rather than have to then deal with a file in application code, especially for specific circumstances of not being allowed to download files to disk.

**Comment by luca-matei** [comment]
I would like to work on this issue.
